components/sidebar/sidebar_category_list/sidebar_category_list.tsx

```diff
@@ -330,6 +349,62 @@ export default class SidebarCategoryList extends React.PureComponent<Props, Stat
         this.updateUnreadIndicators();
     }, 100);
 
+    onBeforeCapture = (before: BeforeCapture) => {
+        // // Ensure no channels are animating
+        this.channelRefs.forEach((ref) => ref.classList.remove('animating'));
+
+        // TODO: Refactor into better code
+        // Turn off scrolling temporarily so that dimensions can be captured
+        const droppable = [...document.querySelectorAll<HTMLDivElement>('[data-rbd-droppable-id*="droppable-categories"]')];
```

Comment:
We'll definitely want automated tests for this since it seems like we're reaching into the library's internals here

Reply:
Agreed, made a ticket: https://mattermost.atlassian.net/browse/MM-24439